### PR TITLE
docs(marketplace): init section from Add-on provider API page

### PR DESCRIPTION
**Pull request by davlgd**
## Describe your PR

Following new Marketplace strategy and after discussions with Alliance team, I've moved the Add-on provider section to a 1st level for Marketplace and started to fix content. Most part of the page will be rewritten later after deeper inspection/tests.

### CODE REVIEW

**Comment by allemas** [comment]
Looks good to me, if necessary, we'll make further modifications
Thanks @davlgd for this !

**Review by allemas — APPROVED** [review]
Looks good to me, if necessary, we'll make further modifications

File: content/doc/marketplace/_index.md [hunk 1]
```diff
@@ -66,66 +49,29 @@ Here is what the manifest JSON must look like:
 
 Fields
 
-* `id` - An id for your add-on. All lower case, no spaces or punctuation. Underscores and dashes are allowed. This can’t be changed after the first push. It is also used for HTTP basic auth when making provisioning calls.
+* `id` - An ID for your add-on. All lower case, no spaces or punctuation. Underscores and dashes are allowed. This can’t be changed after the first push. It is also used for HTTP basic auth when making provisioning calls.
 
-* `name` (Optional) - A human readable name for your add-on. You will be able to change it later in the dashboard, so you don't even have to provide it right now.
+* `name` (Optional) - An human-readable name for your add-on. You will be able to change it later in the dashboard, so you don't even have to provide it right now.
```
Comment (juliamrch) [inline content/doc/marketplace/_index.md:54]:
```suggestion
* `name` (Optional) - A human-readable name for your add-on. You will be able to change it later in the dashboard, so you don't even have to provide it right now.
```

File: content/doc/marketplace/_index.md [hunk 2]
```diff
@@ -66,66 +49,29 @@ Here is what the manifest JSON must look like:
...
 
-* `api/config_vars` - A list of config vars that will be returned on provisioning calls. Each config_var name *MUST* start with the capitalized, underscorized add-on id, as in the example.
+* `api/config_vars` - A list of config vars that will be returned on provisioning calls. Each `config_var` name *MUST* start with the capitalized, add-on id with underscores, as in the example.
```
Comment (juliamrch) [inline content/doc/marketplace/_index.md:56]:
```suggestion
* `api/config_vars` - A list of configuration variables that will be returned on provisioning calls. Each `config_var` name must start with the capitalized, add-on id with underscores, as in the example.
```

What does this sentence even mean? "Must be written in capital letters", maybe?

File: content/doc/marketplace/_index.md [hunk 3]
```diff
@@ -66,66 +49,29 @@ Here is what the manifest JSON must look like:
...
 
 * `api/password` - Password that Clever Cloud will send in HTTP basic auth when making provisioning calls. You should generate a long random string for that field.
 
 * `api/sso_salt` - Shared secret used in single sign-on between the Clever Cloud admin panel and your service’s admin panel. You should generate a long random string for that field.
 
 * `api/regions` - The list of geographical zones supported by your add-on. It cannot be empty. As for now, it *MUST* contain the element "eu". More will be supported.
 
-* `api/production/base_url` - The production endpoint for Clever Cloud api actions (provision, deprovision, and plan change).
+* `api/production/base_url` - The production endpoint on which Clever Cloud will send actions requests (provision, deprovision, and plan change).
```
Comment (juliamrch) [inline content/doc/marketplace/_index.md:64]:
```suggestion
* `api/production/base_url` - The production endpoint on which Clever Cloud sends actions requests (provision, deprovision, and plan change).
```

File: content/doc/marketplace/_index.md [hunk 4]
```diff
@@ -66,66 +49,29 @@ Here is what the manifest JSON must look like:
...
 
 * `api/production/sso_url` - The production endpoint for single sign-on.
 
-* `api/test/base_url` - The test endpoint for Clever Cloud api actions. Used to test your service when you create an add-on provider. Then it is the `api/production/sso_url` which is used.
-
-* `api/test/sso_url` - The test endpoint for single sign-on. Used to test your service when you create an add-on provider. Then it is the `api/production/base_url` which is used.
+* `api/test/base_url` - The test endpoint on which Clever Cloud will send actions requests. Used to test your service when you create an add-on provider. Then it is the `api/production/base_url` which is used.
```
Comment (juliamrch) [inline content/doc/marketplace/_index.md:68]:
```suggestion
* `api/test/base_url` - The test endpoint on which Clever Cloud sends actions requests. Used to test your service when you create an add-on provider. After the add-on creation,`api/production/base_url` is used.
```

Unclear sentence. Maybe rework this later?

File: content/doc/marketplace/_index.md [hunk 5]
```diff
@@ -66,66 +49,29 @@ Here is what the manifest JSON must look like:
...
 
-### Authentication
+* `api/test/sso_url` - The test endpoint for single sign-on. Used to test your service when you create an add-on provider. Then it is the `api/production/sso_url` which is used.
```
Comment (juliamrch) [inline content/doc/marketplace/_index.md:70]:
```suggestion
* `api/test/sso_url` - The test endpoint for single sign-on. Used to test your service when you create an add-on provider. After that, the `api/production/sso_url` is used.
```

File: content/doc/marketplace/_index.md [hunk 6]
```diff
@@ -388,11 +267,7 @@ This endpoint gives you more informations about a provisioned add-on.
 
 * `domains` - Originally the domains names for the application owning the add-on. We return an empty list.
 
-#### Update the config vars for an add-on
-
-**Caution**: this endpoint is **not** available during the provisioning call. If you want
-informations, you need to reply to the provisioning call, **then** you can call this
-endpoint.
+### Update the config vars for an add-on
```
Comment (juliamrch) [inline content/doc/marketplace/_index.md:270]:
```suggestion
### Update the configuration variables for an add-on
```

File: content/doc/_index.md [hunk 7]
```diff
@@ -17,10 +17,9 @@ linkTitle: Documentation
   {{< card link="../doc/applications" title="Deploy" subtitle="Deploy any app on Clever Cloud." icon="rocket-launch" >}}
    {{< card link="../doc/addons" title="Connect an add-on" subtitle="Connect a managed database and set up cloud storage." icon="plug" >}}
   {{< card link="../doc/find-help" title="Find help" subtitle="Lost? See how to get help." icon="support" >}}
```
Comment (juliamrch) [inline content/doc/_index.md:19]:
```suggestion
  {{< card link="../doc/find-help" title="Find help" subtitle="Lost? See how to get help." icon="support" >}}
  {{< card link="../doc/marketplace" title="Marketplace" subtitle="Integrate your own service on Clever Cloud" icon="puzzle" >}}
```

File: content/doc/marketplace/_index.md [hunk 8]
```diff
@@ -1,47 +1,30 @@
...
-
-This is the API *you* need to provide to allow Clever Cloud to provision an add-on for a customer.
-
-### How it works
-
-When a Clever Cloud customer uses the marketplace to provision an add-on, Clever Cloud sends a request to your service which provides a new add-on for this app.
+---
 
-### Build an add-on
+Clever Cloud allows its Marketplace partners to provide services as add-ons with revenue sharing. Thus, they can be available to purchase and provision from the [Console](https://console.clever-cloud.com), [Clever Tools](https://github.com/CleverCloud/clever-tools), the [API](/api) or other integrations such as the [Terraform provider](https://registry.terraform.io/providers/CleverCloud/clevercloud/).
```
Comment (juliamrch) [inline content/doc/marketplace/_index.md:17]:
```suggestion
{< hextra/hero-subtitle >}} Clever Cloud allows its Marketplace partners to provide services as add-ons with revenue sharing. Thus, they can be available to purchase and provision from the [Console](https://console.clever-cloud.com), [Clever Tools](https://github.com/CleverCloud/clever-tools), the [API](/api) or other integrations such as the [Terraform provider](https://registry.terraform.io/providers/CleverCloud/clevercloud/).{{< /hextra/hero-subtitle >}}

```

**Review by juliamrch — CHANGES_REQUESTED** [review]
Thank you for this very much needed update on the marketplace integration 🙏 I've added some suggestions to increase clarity on the relevant parts of the files, as well as a card on the documentation index. Feel free to discuss the suggestions if needed.

File: content/doc/marketplace/_index.md (hunk 8 above)
Comment (davlgd) [reply content/doc/marketplace/_index.md:17]: I prefer not to rely on hextra specific templating when not necessary

**Review by juliamrch — APPROVED** [review]
Good to go!

**Pull request merged**